app/components/data-table-row-previous.js

```diff
@@ -33,7 +33,7 @@ export default Component.extend({
 
   noPriorData: computed('data.previous.sum', function() {
     const { 'data.previous': previous } = this.getProperties('data.previous');
-    if ( previous && typeof previous.sum === 'undefined') {
+    if (previous && (typeof previous.sum === 'undefined' || previous.sum === null)) {
```

Comment:
We should merge this and rely on POs to find regressions but just flagging this as the sort of thing that will be a good place to start if they fine any.